requirements-test.py3.txt

```diff
@@ -0,0 +1,7 @@
+django-mptt~=0.11.0
+mock~=3.0.0
```

Comment:
`mock` is included in the stdlib in Python 3, are there API differences being used?